tests/debuginfo/numeric-types.rs

```diff
@@ -203,40 +203,42 @@
 // lldb-command:run
 
 // lldb-command:print/d nz_i8
-// lldb-check:[...]$0 = 11 { __0 = 11 }
+// lldb-check:[...]$0 = 11 { __0 = { 0 = 11 } }
 
 // lldb-command:print nz_i16
-// lldb-check:[...]$1 = 22 { __0 = 22 }
+// lldb-check:[...]$1 = 22 { __0 = { 0 = 22 } }
 
 // lldb-command:print nz_i32
-// lldb-check:[...]$2 = 33 { __0 = 33 }
+// lldb-check:[...]$2 = 33 { __0 = { 0 = 33 } }
 
 // lldb-command:print nz_i64
-// lldb-check:[...]$3 = 44 { __0 = 44 }
+// lldb-check:[...]$3 = 44 { __0 = { 0 = 44 } }
 
 // lldb-command:print nz_i128
-// lldb-check:[...]$4 = 55 { __0 = 55 }
+// lldb-check:[...]$4 = 55 { __0 = { 0 = 55 } }
 
 // lldb-command:print nz_isize
-// lldb-check:[...]$5 = 66 { __0 = 66 }
+// FIXME: `lldb_lookup.summary_lookup` is never called for `NonZero<isize>` for some reason.
```

Comment:
@oli-obk, I got as far as figuring this out, but no idea why it isn't called for `NonZero<isize>` and `NonZero<usize>`.